[vtk] build failure

**Host Environment**
- OS: [Windows]
- Compiler: cl

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install vcpkg install vtk[all]:x64-windows`

**Failure logs** 
Console output:

```
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: ninja -v
    Working Directory: C:/vcpkg/buildtrees/vtk/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\vcpkg\buildtrees\vtk\config-x64-windows-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:300 (vcpkg_execute_required_process)
  ports/vtk/portfile.cmake:135 (vcpkg_configure_cmake)
  scripts/ports.cmake:79 (include)

Error: Building package vtk:x64-windows failed with: BUILD_FAILED
```

In "C:\vcpkg\buildtrees\vtk\config-x64-windows-out.log":
```
[1/2] cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-173cb3e94f.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=OFF" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_MPI=YES" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows""
FAILED: ../CMakeCache.txt 
cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-173cb3e94f.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=OFF" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_MPI=YES" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows""
-- The C compiler identification is MSVC 19.27.29110.0
-- The CXX compiler identification is MSVC 19.27.29110.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test Support for 64 bit file systems
-- Performing Test Support for 64 bit file systems - Failed
-- Module testing disabled due to `VTK_FORBID_DOWNLOADS` being set.
CMake Error at CMake/vtkModule.cmake:959 (message):
  The VTK::IOParallelNetCDF module requires the disabled module VTK::mpi.
Call Stack (most recent call first):
  CMakeLists.txt:270 (vtk_module_scan)


-- Configuring incomplete, errors occurred!
See also "C:/vcpkg/buildtrees/vtk/x64-windows-rel/CMakeFiles/CMakeOutput.log".
See also "C:/vcpkg/buildtrees/vtk/x64-windows-rel/CMakeFiles/CMakeError.log".
[2/2] cmd /c "cd ../../x64-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-173cb3e94f.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=OFF" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_MPI=YES" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows/debug""
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
cmd /c "cd ../../x64-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-173cb3e94f.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=OFF" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_MPI=YES" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows/debug""
-- The C compiler identification is MSVC 19.27.29110.0
-- The CXX compiler identification is MSVC 19.27.29110.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test Support for 64 bit file systems
-- Performing Test Support for 64 bit file systems - Failed
-- Module testing disabled due to `VTK_FORBID_DOWNLOADS` being set.
CMake Error at CMake/vtkModule.cmake:959 (message):
  The VTK::IOParallelNetCDF module requires the disabled module VTK::mpi.
Call Stack (most recent call first):
  CMakeLists.txt:270 (vtk_module_scan)


-- Configuring incomplete, errors occurred!
See also "C:/vcpkg/buildtrees/vtk/x64-windows-dbg/CMakeFiles/CMakeOutput.log".
See also "C:/vcpkg/buildtrees/vtk/x64-windows-dbg/CMakeFiles/CMakeError.log".
ninja: build stopped: subcommand failed.

```

I realized the error above was showing after I set `-DVTK_FORBID_DOWNLOADS=ON`. And I did that because my first built failed with the error:

```
[1/2] cmd /c "cd ../../x64-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=ON" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_Qt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQtSQL=YES" "-DVTK_MODULE_ENABLE_VTK_RenderingQt=YES" "-DVTK_MODULE_ENABLE_VTK_ViewsQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportMFC=YES" "-DVTK_WRAP_PYTHON=ON" "-DVTK_PYTHON_VERSION=3" "-DPython3_FIND_REGISTRY=NEVER" "-DPython3_EXECUTABLE:PATH=C:/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe" "-DVTK_GROUP_ENABLE_MPI=YES" "-DVTK_USE_TK=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/ninja-1.10.0/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows/debug""
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
cmd /c "cd ../../x64-windows-dbg && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=ON" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_Qt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQtSQL=YES" "-DVTK_MODULE_ENABLE_VTK_RenderingQt=YES" "-DVTK_MODULE_ENABLE_VTK_ViewsQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportMFC=YES" "-DVTK_WRAP_PYTHON=ON" "-DVTK_PYTHON_VERSION=3" "-DPython3_FIND_REGISTRY=NEVER" "-DPython3_EXECUTABLE:PATH=C:/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe" "-DVTK_GROUP_ENABLE_MPI=YES" "-DVTK_USE_TK=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/ninja-1.10.0/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows/debug""
-- The C compiler identification is MSVC 19.27.29111.0
-- The CXX compiler identification is MSVC 19.27.29111.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_MomentInvariants when VTK_FORBID_DOWNLOADS is
  ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/MomentInvariants.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


-- Found Git: C:/Program Files/Git/cmd/git.exe (found version "2.28.0.windows.1") 
fatal: Needed a single revision
CMake Error at CMake/vtkModuleRemote.cmake:64 (message):
  Failed to get the hash for
  https://gitlab.kitware.com/vtk/MomentInvariants.git HEAD
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:110 (_git_update)
  CMake/vtkModuleRemote.cmake:157 (_fetch_with_git)
  Remote/MomentInvariants.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


-- Configuring incomplete, errors occurred!
See also "C:/vcpkg/buildtrees/vtk/x64-windows-dbg/CMakeFiles/CMakeOutput.log".
[2/2] cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=ON" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_Qt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQtSQL=YES" "-DVTK_MODULE_ENABLE_VTK_RenderingQt=YES" "-DVTK_MODULE_ENABLE_VTK_ViewsQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportMFC=YES" "-DVTK_WRAP_PYTHON=ON" "-DVTK_PYTHON_VERSION=3" "-DPython3_FIND_REGISTRY=NEVER" "-DPython3_EXECUTABLE:PATH=C:/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe" "-DVTK_GROUP_ENABLE_MPI=YES" "-DVTK_USE_TK=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/ninja-1.10.0/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows""
FAILED: ../CMakeCache.txt 
cmd /c "cd .. && "C:/Program Files/CMake/bin/cmake.exe" "C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean" "-DVTK_USE_CUDA=OFF" "-DVTK_BUILD_ALL_MODULES=ON" "-DBUILD_TESTING=OFF" "-DVTK_BUILD_TESTING=OFF" "-DVTK_BUILD_EXAMPLES=OFF" "-DVTK_INSTALL_INCLUDE_DIR=include" "-DVTK_INSTALL_DATA_DIR=share/vtk/data" "-DVTK_INSTALL_DOC_DIR=share/vtk/doc" "-DVTK_INSTALL_PACKAGE_DIR=share/vtk" "-DVTK_INSTALL_RUNTIME_DIR=bin" "-DVTK_FORBID_DOWNLOADS=ON" "-DVTK_ENABLE_REMOTE_MODULES=OFF" "-DVTK_GROUP_ENABLE_StandAlone=YES" "-DVTK_GROUP_ENABLE_Rendering=YES" "-DVTK_GROUP_ENABLE_Views=YES" "-DVTK_USE_TK=OFF" "-DVTK_USE_EXTERNAL:BOOL=ON" "-DVTK_MODULE_USE_EXTERNAL_VTK_gl2ps:BOOL=OFF" "-DVTK_GROUP_ENABLE_Qt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportQtSQL=YES" "-DVTK_MODULE_ENABLE_VTK_RenderingQt=YES" "-DVTK_MODULE_ENABLE_VTK_ViewsQt=YES" "-DVTK_MODULE_ENABLE_VTK_GUISupportMFC=YES" "-DVTK_WRAP_PYTHON=ON" "-DVTK_PYTHON_VERSION=3" "-DPython3_FIND_REGISTRY=NEVER" "-DPython3_EXECUTABLE:PATH=C:/vcpkg/downloads/tools/python/python-3.8.3-x64/python.exe" "-DVTK_GROUP_ENABLE_MPI=YES" "-DVTK_USE_TK=OFF" "-DCMAKE_MAKE_PROGRAM=C:/vcpkg/downloads/tools/ninja/ninja-1.10.0/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/vcpkg/installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/vcpkg/packages/vtk_x64-windows""
-- The C compiler identification is MSVC 19.27.29111.0
-- The CXX compiler identification is MSVC 19.27.29111.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_MomentInvariants when VTK_FORBID_DOWNLOADS is
  ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/MomentInvariants.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


-- Found Git: C:/Program Files/Git/cmd/git.exe (found version "2.28.0.windows.1") 
--  The remote module: https://gitlab.kitware.com/vtk/MomentInvariants.git is cloned into the directory C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean/Remote/MomentInvariants
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_PoissonReconstruction when
  VTK_FORBID_DOWNLOADS is ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/PoissonReconstruction.remote.cmake:4 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


--  The remote module: https://github.com/lorensen/PoissonReconstruction is cloned into the directory C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean/Remote/PoissonReconstruction
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_Powercrust when VTK_FORBID_DOWNLOADS is ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/Powercrust.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


--  The remote module: https://github.com/lorensen/Powercrust is cloned into the directory C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean/Remote/Powercrust
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_SignedTensor when VTK_FORBID_DOWNLOADS is ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/SignedTensor.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


--  The remote module: https://github.com/KitwareMedical/VTKSignedTensor is cloned into the directory C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean/Remote/SignedTensor
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_SplineDrivenImageSlicer when
  VTK_FORBID_DOWNLOADS is ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/SplineDrivenImageSlicer.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


--  The remote module: https://github.com/martinken/midas-journal-838.git is cloned into the directory C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean/Remote/SplineDrivenImageSlicer
CMake Error at CMake/vtkDownload.cmake:8 (message):
  Attempted to download Module_vtkDICOM when VTK_FORBID_DOWNLOADS is ON
Call Stack (most recent call first):
  CMake/vtkModuleRemote.cmake:142 (vtk_download_attempt_check)
  Remote/vtkDICOM.remote.cmake:5 (vtk_fetch_module)
  Remote/CMakeLists.txt:9 (include)


--  The remote module: https://github.com/dgobbi/vtk-dicom is cloned into the directory C:/vcpkg/buildtrees/vtk/src/ad8774e85e-533405c692.clean/Remote/vtkDICOM
-- Performing Test Support for 64 bit file systems
-- Performing Test Support for 64 bit file systems - Failed
-- Module testing disabled due to `VTK_FORBID_DOWNLOADS` being set.
CMake Error at CMake/vtkModule.cmake:959 (message):
  The VTK::ParallelMPI4Py module requires the disabled module VTK::mpi.
Call Stack (most recent call first):
  CMakeLists.txt:270 (vtk_module_scan)


-- Configuring incomplete, errors occurred!
See also "C:/vcpkg/buildtrees/vtk/x64-windows-rel/CMakeFiles/CMakeOutput.log".
See also "C:/vcpkg/buildtrees/vtk/x64-windows-rel/CMakeFiles/CMakeError.log".
ninja: build stopped: subcommand failed.
```

--- a/ports/vtk/CONTROL
+++ b/ports/vtk/CONTROL
@@ -1,6 +1,6 @@
 Source: vtk
 Version: 9.0.1
-Port-Version: 1
+Port-Version: 2
 Description: Software system for 3D computer graphics, image processing, and visualization
 Homepage: https://github.com/Kitware/VTK
 Build-Depends: zlib, libpng, tiff, libxml2, jsoncpp, glew, freetype, expat, hdf5[core], libjpeg-turbo, proj4, lz4, liblzma, libtheora, eigen3, double-conversion, pugixml, libharu[notiffsymbols], sqlite3, netcdf-c, utfcpp, libogg, pegtl-2

--- a/ports/vtk/portfile.cmake
+++ b/ports/vtk/portfile.cmake
@@ -68,6 +68,7 @@ endif()
 if("mpi" IN_LIST FEATURES)
     list(APPEND ADDITIONAL_OPTIONS
         -DVTK_GROUP_ENABLE_MPI=YES
+        -DVTK_USE_MPI=YES
     )
 endif()
 

--- a/ports/vtk/portfile.cmake
+++ b/ports/vtk/portfile.cmake
@@ -72,6 +72,12 @@ if("mpi" IN_LIST FEATURES)
     )
 endif()
 
+if("mpi" IN_LIST FEATURES AND "python" IN_LIST FEATURES)
+    list(APPEND ADDITIONAL_OPTIONS
+        -DVTK_MODULE_USE_EXTERNAL_VTK_mpi4py=OFF
+    )
+endif()
+
 if("opengl" IN_LIST FEATURES)
     list(APPEND ADDITIONAL_OPTIONS
         -DVTK_MODULE_ENABLE_VTK_DomainsChemestryOpenGL2=YES

